tests/Fixtures/FakeEntityManager.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+declare( strict_types = 1 );
+
+namespace WMDE\Fundraising\Frontend\Tests\Fixtures;
+
+use Doctrine\ORM\EntityManager;
+
+class FakeEntityManager extends EntityManager {
```

Comment:
In our [nomenclature of test doubles](https://www.entropywins.wtf/blog/2016/05/13/5-ways-to-write-better-mocks/) this would be an `EntityManagerSpy`, because it records calls.